dist/debian/debian/rules

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/make -f
+
+include /usr/share/dpkg/pkg-info.mk
+
+export PYBUILD_DISABLE=1
+jobs := $(shell echo $$DEB_BUILD_OPTIONS | sed -r "s/.*parallel=([0-9]+).*/-j\1/")
+ifneq ($(findstring housekeeping, $(DEB_BUILD_OPTIONS)),)
+    install_arg := --housekeeping
+else
+    install_arg :=
+endif
+product := $(shell cat SCYLLA-PRODUCT-FILE)
+version := $(shell cat SCYLLA-VERSION-FILE)
+release := $(shell cat SCYLLA-RELEASE-FILE)
+distribution := stable
+fullname := ScyllaDB Inc.
+email := info@scylladb.com
+logmsg := Release $(product)-$(version).
+
+clean:
+ifneq ($(product),scylla)
+	sed -e 's/scylla-server/$(product)-server/g' \
+		-e 's/scylla-conf/$(product)-conf/g' \
+		-e 's/scylla-kernel-conf/$(product)-kernel-conf/g' \
+		-e 's/scylla-python3/$(product)-python3/g' \
+		-e 's/scylla-jmx/$(product)-jmx/g' \
+		-e 's/scylla-tools/$(product)-tools/g' \
+		-e 's/scylla-tools-core/$(product)-tools-core/g' \
+		-e 's/Package: scylla$$/Package: $(product)/g' debian/control > debian/control.tmp
+	mv debian/control.tmp debian/control
```

Comment:
Looks like I gave you bad advice. This is templating in sed. I thought we only needed to template `rules`, but I was wrong.

If we have to do this, then I prefer the first patch. Can you post a link to it so I can look at it again?